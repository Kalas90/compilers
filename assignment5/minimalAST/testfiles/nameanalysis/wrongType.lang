int main() {
	int a;
	int b;
	a = a < b;
	b = a * c;
	b = c * a;
}
