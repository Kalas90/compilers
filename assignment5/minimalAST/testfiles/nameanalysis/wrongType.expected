Error at line 4: Symbol was <boolean> but expected <int>
Error at line 5: symbol 'c' is not declared
Error at line 6: symbol 'c' is not declared
